scripts/common

```diff
@@ -1,6 +1,24 @@
 #!/usr/bin/env bash
-set -ex
+set -exu
 
 # Export env vars needed for running test dependencies
 export ORG=mrcide
-export OUTPACK_SERVER_TAG=$ORG/outpack_server:main
+export OUTPACK_SERVER_TAG=$ORG/outpack_server:497d8b1
+
+export CONTAINER_NAMESPACE=orderly.runner
+
+export NETWORK=$CONTAINER_NAMESPACE-network
+export REDIS=$CONTAINER_NAMESPACE-redis
+export DEBUG=$CONTAINER_NAMESPACE-debug
```

Comment:
So this is the docker container where you're going to do your git setup? I find `DEBUG` quite a confusing suffix for that!
```suggestion
export GIT_SETUP=$CONTAINER_NAMESPACE-git-setup
```

Reply:
done